### PR TITLE
Add org.gnome.gitlab.cheywood.Pulp

### PR DESCRIPTION
**Pull request by cheywood**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.

Pulp is a feed reader for GNOME desktop and mobile devices written with an interactive focus of parsing efficiently through an excessive amount of feeds and articles, aiming to frequently rest in an empty / all read state. Like an empty inbox, the antithesis of doom scrolling.

This represents the first release as a preview which only currently supports sync via FreshRSS; there is no direct RSS access right now.

- [ ] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

edit: typo

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.gnome.gitlab.cheywood.Pulp.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "pulp",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.secrets"
```
Comment (bbhtt) [inline org.gnome.gitlab.cheywood.Pulp.json:13]: This does not work over the portal?

**Comment by bbhtt** [comment]
You might need to lowercase the last component of the ID if the linter complains. Gitlab namespaces are case sensitive, see the control over domain portion https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain

**Comment by bbhtt** [comment]
bot, build org.gnome.gitlab.cheywood.Pulp

**Comment by bbhtt** [comment]
Yea either lowercase the repo name or lowercase the appid.

**Comment by cheywood** [comment]
> Yea either lowercase the repo name in gitlab settings or lowercase the last component of appid.

Oops, right, thanks for that. I'll get onto it. That must have changed in the last few months?

Do I need to make a new submission/MR or I can just change the appstream etc?

**Comment by bbhtt** [comment]
Yea, we started automatically calculating the domain and url from the rDNS-ed appid, then checking if a webpage or a repository is alive on the URL. This affects new submissions only.

> Do I need to make a new submission/MR or I can just change the appstream etc?

No just change it here.

**Comment by cheywood** [comment]
> No just change it here.

Ok thanks, will do :pray:

**Comment by cheywood** [comment]
@bbhtt that should be what we're after :crossed_fingers:

**Comment by bbhtt** [comment]
bot, build org.gnome.gitlab.cheywood.Pulp

File: org.gnome.gitlab.cheywood.Pulp.json (hunk 1 above)
Comment (cheywood) [reply org.gnome.gitlab.cheywood.Pulp.json:13]: I'm not sure? Adding secrets resolved an issue in the past on another project when running on another desktop (non-GNOME).

Are you talking about the others too?

Comment (cheywood) [reply org.gnome.gitlab.cheywood.Pulp.json:13]: > an issue in the past on another project

FYI https://gitlab.gnome.org/World/iotas/-/issues/106

Comment (bbhtt) [reply org.gnome.gitlab.cheywood.Pulp.json:13]: That should be resolved from KDE 6 onwards.

Comment (bbhtt) [reply org.gnome.gitlab.cheywood.Pulp.json:13]: Ah nvm it's a different issue.

**Comment by bbhtt** [comment]
The window not being drag-able is intentional?


https://github.com/user-attachments/assets/5ab8487a-f2fb-4838-96b8-8c6cec0c2de3

**Comment by cheywood** [comment]
> The window not being drag-able is intentional?

Oops, no the first start screen not being dragable wasn't on purpose, thanks for that :+1: Fixed.

The window was dragable once logged in fwiw.

**Comment by bbhtt** [comment]
bot, build org.gnome.gitlab.cheywood.Pulp

**Pull request closed without merging**